cypress/integration/admin/login/login.spec.js

```diff
@@ -17,4 +19,146 @@ context("Login Page", () => {
             cy.findByText(Cypress.env("DEFAULT_ADMIN_USER_USERNAME")).should("exist");
         });
     });
+
+    it("try to access admin pages as anonymous user", () => {
+        cy.visit("/page-builder/pages");
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+
+        cy.visit("/form-builder/forms");
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+
+        cy.visit("/cms/content-models");
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+    });
+
+    it("try to access non-existing page as logged user", () => {
+        cy.visit("/");
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+
+        cy.findByLabelText(/your e-mail/i).type("admin@webiny.com");
+        cy.findByLabelText(/your password/i).type("12345678");
+        cy.findByText(/submit/i).click();
+
+        cy.findByPlaceholderText(/search.../i).should("exist");
+        cy.findByText(/pages/i).should("exist");
+
+        cy.visit("/page-builder/pagessss");
+        cy.findByText(/The route is either missing, or you're not authorized to view it./i).should(
+            "exist"
+        );
+        cy.findByText(/Please contact your administrator to report the issue./i).should("exist");
+        cy.findByText(/Take me back./i).should("exist");
+    });
+
+    it("authenticated user must logout successfully", () => {
+        cy.visit("/");
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
+
+        cy.findByLabelText(/your e-mail/i).type("admin@webiny.com");
+        cy.findByLabelText(/your password/i).type("12345678");
+        cy.findByText(/submit/i).click();
+
+        cy.findByPlaceholderText(/search.../i).should("exist");
+        cy.findByText(/pages/i).should("exist");
+
+        cy.findByTestId("logged-in-user-menu-avatar").click();
+        cy.findByText(/Sign out/i).click();
+        cy.findByText(/sign in/i).should("exist");
+        cy.findByText(/forgot password?/i).should("exist");
```

Comment:
The above code tells us that the user was redirected to the login page, but it doesn't really tell us if the log out worked.

So, what I'm thinking... can we also visit an admin URL and ensure the login is shown instead of the actual page? Basically we can just copy those couple of steps from a previous test, for example:


``` 
cy.visit("/page-builder/pages");
        cy.findByText(/sign in/i).should("exist");
        cy.findByText(/forgot password?/i).should("exist");```